docs/README.md

```diff
@@ -0,0 +1,12 @@
+# nf-core/neutronstar: Documentation
+
+The nf-core/neutronstar documentation is split into the following files:
+
+1. [Installation](https://nf-co.re/usage/installation)
+2. Pipeline configuration
+    * [Local installation](https://nf-co.re/usage/local_installation)
+    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
+    * [Reference genomes](https://nf-co.re/usage/reference_genomes)
```

Comment:
I think you can delete this line as the pipeline doesn't deal with reference genomes.